Tests/PluginGeneratorTests/JarBuilderTests.cs

```diff
@@ -85,7 +85,7 @@ public void JarBuilder_Build()
             Assert.IsTrue(success, "Failed to build the jar file");
             
             new ZipFileChecker(this.TestContext, finalJarPath)
-                .AssertZipContainsFiles(
+                .AssertZipContainsOnlyExpectedFiles(
```

Comment:
Miscellaneous: the jar builder tests also checked the zip file. Turned the loose check into a strict check.